eslint.config.mjs

```diff
@@ -54,12 +54,7 @@ const tsConfig = {
       ecmaFeatures: {
         jsx: true,
       },
-      project: [
-        "./tsconfig.json",
-        "./shared/tsconfig.json",
-        "./vscode/tsconfig.json",
-        "./webview-ui/tsconfig.json",
-      ],
+      project: true,
```

Comment:
Should this be necessary with latest test?

Reply:
hmm... after reverting I see the problem again (on my local repo). Only freshly cloned works for me.

Reply:
I need to use either `project: true` or `projectService: true,`

Reply:
Ah I was able to reproduce this after installing the proper eslint extensions.